### PR TITLE
Expand Candidate Info

### LINKED ISSUE #108: ℹ Expand Candidate Info
**Type**: client
**FOF**:  2

## Description
- Expand the candidate bio page to display extra information from the `/candidate` endpoint
- Extra Content Includes:
    - Issues
    - Experience
    - Education
    - Endorsements

## Screenshot
image

## Todo

- [ ] Display additional data on the candidate bio page

## Links
- [Figma](https://www.figma.com/file/6aJ3cfu2Yq5dxUQcNDkm6Y/%F0%9F%8E%A8-Brink-Designs?node-id=565%3A1390)

### PR DESCRIPTION
**Pull request by ericcccsliu**
## Status:

:rocket: Ready

## Summary

Closes #108

## Changes

- Populate candidate bio with data from BallotReady
- Refactor candidate bio to be more mobile-friendly

## Screenshots
Screen Shot 2022-04-29 at 8 16 38 PM

Screen Shot 2022-04-29 at 8 16 48 PM

### CODE REVIEW

File: client/src/components/link/SocialLink.tsx [hunk 1]
```diff
@@ -34,31 +35,36 @@ const urlPairs = {
...
-      <Text marginLeft="5px" /* hardcoded font size */>
-        {urlPairs[candidateUrl.type] && urlPairs[candidateUrl.type].label}
-      </Text>
-    </Flex>
-  </Link>
-);
+}): ReactElement => {
+  const isMobile = useIsMobile();
+  return (
+    <Link href={candidateUrl.url} target="_blank" rel="noopener" {...rest}>
```
Comment (amit-sawhney) [inline client/src/components/link/SocialLink.tsx:41]: you don't need this stuf if you say isExternal

File: client/src/components/link/SocialLink.tsx [hunk 2]
```diff
@@ -34,31 +35,36 @@ const urlPairs = {
...
+      <Flex
+        direction="row"
+        justify="flex-start"
+        align="center"
+        width={[isMobile ? '55px' : '165px']}
```
Comment (amit-sawhney) [inline client/src/components/link/SocialLink.tsx:46]: Use chakra responsive styling

File: client/src/pages/CandidateBio.tsx [hunk 3]
```diff
@@ -92,37 +105,103 @@ export const CandidateBio = (): ReactElement => {
...
+                    <ListItem key={index}>
+                      {experience.position}
+                      {experience.company && `, ${experience.company}`}
+                      {experience.start_year && `, ${experience.start_year}`}
+                      {experience.end_year && `–${experience.end_year}`}
+                    </ListItem>
+                  ),
+              )}
+            </UnorderedList>
+            {/* {fullCandidate?.endorsements.length && <Text
```
Comment (amit-sawhney) [inline client/src/pages/CandidateBio.tsx:189]: take out unused code

**Review by amit-sawhney — APPROVED** [review]
Fix the stuff we talked about verbally and these comments

File: client/src/pages/tab/BallotCandidates.tsx [hunk 4]
```diff
@@ -25,7 +25,11 @@ export const Candidates = (): ReactElement => {
     (position) => position.name.toLowerCase() === parsedName,
   )?.candidates;
 
-  return (
+  return isLoading ? (
+    <Center height="80vh">
+      <Spinner size="lg" />
+    </Center>
+  ) : (
```
Comment (ayanmallik) [inline client/src/pages/tab/BallotCandidates.tsx:32]: Thanks for adding in these spinners across the ballot page!

**Review by ayanmallik — APPROVED** [review]
Awesome work, Eric - this new and improved candidate bio page is wonderful!

**Pull request merged**